src/main/resources/db/changelog/changes/02-create-users-table.yaml

```
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-book-table
+      author: samoilenko
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: email
+                  type: VARCHAR(255)
+                  constraints:
+                    unique: true
+                    nullable: false
+              - column:
+                  name: password
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: first_Name
```

same for other fields
```suggestion
                  name: first_name
```